internal/cli/email_templates.go

```diff
@@ -192,12 +192,24 @@ func updateEmailTemplateCmd(cli *cli) *cobra.Command {
 			}
 
 			var oldTemplate *management.EmailTemplate
+			templateExists := true
 			err := ansi.Waiting(func() (err error) {
 				oldTemplate, err = cli.api.EmailTemplate.Read(apiEmailTemplateFor(inputs.Template))
 				return err
 			})
 			if err != nil {
-				return fmt.Errorf("failed to get the email template '%s': %w", inputs.Template, err)
+				mErr, ok := err.(management.Error)
+				if !ok || mErr.Status() != http.StatusNotFound {
+					return fmt.Errorf("failed to get the email template '%s': %w", inputs.Template, err)
+				}
+
+				templateExists = false
+				oldTemplate = &management.EmailTemplate{
+					From:    auth0.String(""),
+					Subject: auth0.String(""),
+					Enabled: auth0.Bool(false),
+					Syntax:  auth0.String("liquid"),
```

Comment:
**Question:** Is it necessary to instantiate with empty values here if the `emailTemplate` gets fully constructed below?

Reply:
Sadly yes 😞 as otherwise this line https://github.com/auth0/auth0-cli/pull/611/files#diff-7c49d78e329a015986302ea95a143d68ca7d933a41514ca964b46bddcd22f31aR215 will panic: